hugo.toml

```diff
@@ -202,7 +202,7 @@ enable = false
 RelPermalink= "communityhealthtoolkit.org"
 
 [related]
-  treshold = 1
+  threshold = 1
```

Comment:
I'm curious about this change. The file name changed and the threshold value is now set which might have unintended consequences. What was the logic behind changing this file name?

Reply:
@deining Do you know?

Reply:
> I'm curious about this change. The file name changed (...)
> What was the logic behind changing this file name?

Have a look [here](https://gohugo.io/getting-started/configuration/#configuration-file):

```
With v0.109.0 and earlier the basename of the site configuration file was config
instead of hugo. You can use either, but should transition to the new naming
convention when practical.
```

Hope this make the motivation for the name change clear.

Reply:
Ok, that's great to know!

I'm still cautious about the typo correction, because it'll go from not applying to applying the threshold. It might be safer to delete it instead of fixing the typo...

Reply:
> Ok, that's great to know!
> 
> I'm still cautious about the typo correction, because it'll go from not applying to applying the threshold. It might be safer to delete it instead of fixing the typo...

Agreed. I removed the line  treshold = 1`, I think this the way to go here.

Reply:
Hey @deining , I have undone the name change and the `treshold` change. I will review the aforementioned changes in [this](https://github.com/medic/cht-docs/issues/1420) issue. Thank you for the suggestion.